### DISCUSSION TITLE
Untracked data warning

### DISCUSSION THREAD

**Discussion by TrueWill** [Kong/insomnia#6961] [Feedback]
There was considerable negative feedback when the scratch pad was removed in a previous version, enough so that it was added back.

Please consider having an option to turn off "You have untracked data on your computer" warnings or anything else that would discourage scratch pad usage.

I migrated to Insomnia from Postman because they disabled their scratch pad. People want truly private data. Please keep this in mind.

Thank you.

**Comment by subnetmarco** [comment]
The "untracked data" is your project data that is not linked to a workspace and it addresses the issue of people not being able to find their project data because detached from the organization ID. It has nothing to do with "metrics tracking" if that's what you imply.

You can find untracked data in the Preferences:

<img width="813" alt="Screenshot 2024-01-03 at 9 29 56 AM" src="https://github.com/Kong/insomnia/assets/964813/8fc87778-5240-46c4-b4f1-e5390a6c891f">

**Comment by TrueWill** [comment]
I'm not implying anything about metrics tracking. I'm saying I have no organization or cloud account, and I have not logged in (nor will I).

**Comment by subnetmarco** [comment]
@TrueWill if you go to "Preferences -> Data" can you show me what you see? Are most of your projects with zero files, or do you have files in them?

**Comment by TrueWill** [comment]
<img width="692" alt="image" src="https://github.com/Kong/insomnia/assets/1709726/adc7f486-2982-4eee-b15e-9b6593f9af02">

I just have a POST and a couple of private environments defined in scratch pad.

**Comment by subnetmarco** [comment]
I asked the team, and they confirmed that Scratch Pad data does **not** show up as untracked data, which I thought was the case. 

We will introduce a "Delete" button to remove untracked projects that the user doesn't care about anymore, because in both your case and my case we may not care about some of that data anymore.

By providing the "Delete" button, you will be able to simply remove the untracked projects and that alert will go away.

**Reply by TrueWill** [reply]
Thank you! 😄
